cogs3/.template_env

```diff
@@ -9,10 +9,10 @@ MAINTENANCE_MODE_IGNORE_ADMIN_SITE=False
 
 ALLOWED_HOSTS=["*"]
 
-SECURE_CONTENT_TYPE_NOSNIFF=True
-SECURE_BROWSER_XSS_FILTER=True
-SESSION_COOKIE_SECURE=True
-CSRF_COOKIE_SECURE=True
+SECURE_CONTENT_TYPE_NOSNIFF=False
+SECURE_BROWSER_XSS_FILTER=False
+SESSION_COOKIE_SECURE=False
+CSRF_COOKIE_SECURE=False
```

Comment:
This is useful for spinning up development instances, but it would be worth adding a comment in the README that the default implementation is insecure, and that these should be set to `True` for bringing up a production instance.